internal/cli/kraft/pkg/pkg.go

```diff
@@ -35,7 +35,7 @@ import (
 )
 
 type PkgOptions struct {
-	Architecture string                    `local:"true" long:"arch" short:"m" usage:"Filter the creation of the package by architecture of known targets"`
```

Comment:
The architecture is actually inferred from the host system. There is a detection check somewhere and that one is used.

I would not set a default here as we can't know for sure, but the text message is good.

Reply:
Thanks!
Already corrected.